main.css

```diff
@@ -94,15 +94,56 @@ footer
     background-color: #061428;
 }
 
-#sidebar
+#mobileMenuIcon
 {
     position: fixed;
+    top: 35px;
+    left: 5px;
+    width: 48px;
+    height: 48px;
+    z-index: 9999;
+    background: url("assets/mobile-ui/mobilemenu.png");
```

Comment:
@vonbearshark The pin iconography is actually from an icon font too, so if we do go the icon font route we'd be pretty well set up for `divIcon` pins in future.